doit/cmd_info.py

```diff
@@ -23,6 +23,16 @@ def my_safe_repr(obj, context, maxlevels, level):
     return pprint._safe_repr(obj, context, maxlevels, level)
 
 
+opt_show_build_reason = {
+    'name': 'show_build_reason',
+    'short': 'R',
+    'long': 'show-build-reason',
```

Comment:
IMO this name is too long. Maybe just `status` would be ok. The short name `R` stands for what? and why a capital letter?